TWINCAT/iocBoot/iocTWINCAT-IOC-01/st-common.lua

```diff
@@ -14,7 +14,7 @@ function twincat_stcommon_main()
 	local ioc_name = ibex_utils.getMacroValue{macro="IOCNAME"}
 	local plc_version = ibex_utils.getMacroValue{macro="PLC_VERSION", default="1"}
 
-	iocsh.tcSetScanRate(150, 2)
+	iocsh.tcSetScanRate(10, 5)
 
 	local full_tpy_path = ibex_utils.getMacroValue{macro="TWINCATCONFIG"} .. "/" .. tpy_file
 	if not exists(full_tpy_path) then
```

Comment:
Nitpick: While this function is being changed, should we use this opportunity to add a descriptive function docstring, maybe starting along the lines of: 
```
 /* 
  * main function for st-common.lua to be called from the IOC
  * at startup.
  */
```
and add a comment to the iocsh.tcSetScanRate(10, 5). Maybe something along the lines of  `-- Set the tcSetScanRate such that BMOVING.RVAL is updated before DMOV`

Reply:
we could, but I think given it's the only function that's called from st-common.cmd I'm not sure a docstring is particularly useful to explaining what the code is doing? maybe more comments in the code itself describing each step would be more helpful. The scan rate isn't just limited to BMOVING so probs best to make it more general

Reply:
I agree, was more or less because I couldn't justify asking you clean up the entire function given that's not the purpose of this issue.
Probably worth creating a new issue to refactor the function into smaller functions and add comments and docstrings etc to clean it up a bit. I will create a new issue for it once I've merged this in.

Reply:
have added some comments documenting what's going on for bits that may be harder to understand at first glance